app/_config/theme/index.ts

```diff
@@ -26,8 +21,8 @@ const theme = extendTheme({
     },
   },
   fonts: {
-    body: inter.style.fontFamily,
-    heading: inter.style.fontFamily,
+    body: acumin.style.fontFamily,
+    heading: rift.style.fontFamily,
```

Comment:
Did this not work - 

```
styles: {
    global: {
      'html, body': {
        color: 'gray.600',
        lineHeight: 'tall',
      },
      a: {
        color: 'teal.500',
      },
    },
  },
```

Like give the fonts definition for body and heading, then assign / override and add one for h1 with a different font. Ref - https://chakra-ui.com/docs/styled-system/global-styles

Reply:
Didn't try that, will give it a go